experimental/terraform/resource_event_handler.tf

```diff
@@ -39,8 +39,8 @@ resource "google_cloud_run_service" "event_handler" {
 }
 
 resource "google_cloud_run_domain_mapping" "event_handler" {
-  location = var.google_region
-  name     = "dora.nandos.dev"
+  location = var.google_domain_mapping_region
```

Comment:
You can make this conditional using `count = length(var.google_domain_mapping_region) > 0 ? 1 : 0`. This will need `google_domain_mapping_region` to have a default of `""`